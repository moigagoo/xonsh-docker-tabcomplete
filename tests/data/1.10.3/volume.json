{
        "arguments": [
                "--help"
        ],
        "commands": [
                "create",
                "inspect",
                "ls",
                "rm"

        ]
}
