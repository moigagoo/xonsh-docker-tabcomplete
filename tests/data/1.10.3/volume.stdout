
Usage:	docker volume [OPTIONS] [COMMAND]

Manage Docker volumes

Commands:
  create                   Create a volume
  inspect                  Return low-level information on a volume
  ls                       List volumes
  rm                       Remove a volume

Run 'docker volume COMMAND --help' for more information on a command

  --help             Print usage
